src/main.cpp

```diff
@@ -62,6 +65,17 @@ static struct
     bool dry_run = false;
 } global_options;
 
+struct formatted_pkg
+{
+    std::string name, version, build, channel;
+};
+
+bool
+compareAlphabetically(const formatted_pkg& a, const formatted_pkg& b)
+{
+    return a.name < b.name;
+}
+
```

Comment:
Ist das ok?
Had to write the struct here to use in this function! :/

Reply:
fine for now :)